.travis.yml

```diff
@@ -2,3 +2,8 @@ language: node_js
 
 node_js:
   - "node"
+
+rvm:
+  - 2.2
+
+script: "bundle exec jekyll build"
```

Comment:
since we still want to use the NPM test target too, mabye add manually add that as a line again here